From: xMeM <haooy@outlook.com>
Subject: [PATCH 1/5] util: build the wrapper with Linux WSI on Android

The wrapper is a Bionic binary, but it needs Mesa's Linux X11/XCB WSI paths
rather than Mesa's Android platform path.

diff --git a/src/util/detect_os.h b/src/util/detect_os.h
index 86286dfbe70..f589104f6a7 100644
--- a/src/util/detect_os.h
+++ b/src/util/detect_os.h
@@ -24,7 +24,7 @@
  * Android defines __linux__, so DETECT_OS_LINUX and DETECT_OS_POSIX will
  * also be defined.
  */
-#if defined(__ANDROID__)
+#if 0
 #define DETECT_OS_ANDROID 1
 #endif
 
